tasks.py

```diff
@@ -223,22 +215,31 @@ def docker_switching_branch(ctx):
     docker_manage(ctx, "block_inventory")
 
 
-@task
-def docker_new_db(ctx):
-    """Delete your database and create a new one with fake data"""
+def docker_stop_and_delete_db(ctx):
     print("Stopping services first")
     ctx.run("docker-compose down")
     print("Deleting database")
-    ctx.run("docker volume rm foundationmozillaorg_postgres_data")
-    print("Applying database migrations.")
-    docker_migrate(ctx)
-    print("Creating fake data")
-    docker_manage(ctx, "load_fake_data")
-    print("Updating localizable fields")
-    docker_l10n_sync(ctx)
-    docker_l10n_update(ctx)
-    print("Updating block information")
-    docker_manage(ctx, "block_inventory")
+    try:
+        ctx.run("docker volume rm foundationmozillaorg_postgres_data")
+    except:
```

Comment:
flake8 doesn't allow a bare except, but I don't know how to find out what the actual error type is: the log doesn't help in this case.